ui/src/app/core/job-manager.service.ts

```diff
@@ -106,7 +107,7 @@ export class JobManagerService {
   }
 
   updateJobLabels(id: string, req: UpdateJobLabelsRequest): Promise<UpdateJobLabelsResponse> {
-    return this.http.post(`${environment.apiUrl}/jobs/${id}/updateLabels`,
+    return this.http.post(`${this.environmentConfigurationLoaderService.getEnvironmentConfigSynchronous()['apiUrl']}/jobs/${id}/updateLabels`,
```

Comment:
same here and below local variable

Reply:
What do you think about declaring an instance variable e.g. `apiUrl`and assign it with `this.environmentConfigurationLoaderService.getEnvironmentConfigSynchronous()['apiUrl']` so that all of the methods can share the value?